### ISSUE TITLE
 UI/UX Improvements Suggestion for Your Project

### ISSUE THREAD

**Issue by Anisha2123** [BamaCharanChhandogi/GitFinder#46]
Hi @BamaCharanChhandogi,

I hope this message finds you well. I've been exploring your project and would love to contribute by enhancing its user interface for a more modern and engaging experience. Here are a few changes I believe would improve the look and usability of the page:

1. Left Sidebar: Shift the left bar to the extreme left for better structure and alignment.
2. Image Hover Effect: Add a hover effect to the image in the left bar, making it pop out when hovered.
3. Box Shadows: Introduce subtle box shadows to give the layout more depth and a cleaner visual hierarchy.
4. Font Styling: Update the font to improve readability and aesthetics, giving it a fresh, professional feel.
5. Button Hover Effects: Add smooth hover transitions on buttons to make them more interactive and user-friendly.

I'd love to help implement these changes to enhance the overall UI/UX. Please let me know if you'd like to move forward with this, and I'd be happy to collaborate!

Best regards,
Anisha Birla

**Comment by BamaCharanChhandogi** [comment]
👋 Thank you for raising an issue! We appreciate your effort in helping us improve. Our GitFinder team will review it shortly. Stay tuned!

**Comment by tmsagarofficial** [comment]
What's the update on this @Anisha2123 .
Feel free to reach out to us if you have any queries or need help with anything. 🚀

**Comment by inkerton** [comment]
Can You assign me this issue ?
@BamaCharanChhandogi

**Comment by Anisha2123** [comment]
Yes Sir, I m working on it. Will update it soon @tmsagarofficial
